packages/react-magma-dom/src/components/AIButton/index.tsx

```diff
@@ -0,0 +1,161 @@
+import * as React from 'react';
+
+import { IconProps } from 'react-magma-icons';
+
+import { useIsInverse } from '../../inverse';
+import { ThemeInterface } from '../../theme/magma';
+import { resolveProps, XOR } from '../../utils';
+import { ButtonGroupContext } from '../ButtonGroup';
+import { StyledAIButton } from './StyledAIButton';
+
+export enum AIButtonVariant {
+  variantA = 1, //default
+  variantB = 2,
```

Comment:
I noticed this is what shows up on storybook:
<img width="462" height="145" alt="image" src="https://github.com/user-attachments/assets/ccda7412-2182-4576-b612-f82adcbee0e0" />

should we swap these values to be consistent? 

```suggestion
  variantA = 'variantA', //default
  variantB = 'variantB',
```